docs/run-azure.md

```diff
@@ -50,8 +49,12 @@ The deployment is also hooked up to the PR process, which will create another en
 ## Environment Removal
 
 **IT IS IMPORTANT TO CLEAR DOWN ANY CREATED ENVIRONMENTS**
-To do so, connect `kubectl` as above, and run:
 
-- `tear-down-azure.sh <namespace>` to remove the deployment
-- Remove any created DBs in the azure database server
-- Remove the created container on the azure storage account
\ No newline at end of file
+*****NOTE*****
+In order for the script to also clear the databases and storage containers, you'll need to be connected to the VPN
+
+Run the tear down script 
+
+`tear-down-azure.sh <namespace> -a <blob store account connection string>`
```

Comment:
shouldn't the connection string be in quotes?

Reply:
do you not need -n for namespace?

Reply:
You do indeed, good point in both cases, thanks!